src/main/java/com/questhelper/panel/AssistLevelPanel.java

```diff
@@ -0,0 +1,196 @@
+
+import com.questhelper.QuestHelperConfig;
+import com.questhelper.QuestHelperPlugin;
+import com.questhelper.config.AssistanceLevel;
+import com.questhelper.managers.QuestManager;
+import com.questhelper.questhelpers.QuestHelper;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.ui.ColorScheme;
+import net.runelite.client.ui.DynamicGridLayout;
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
+
+public class AssistLevelPanel extends JPanel
+{
+	private final FixedWidthPanel mainPanel;
+	private ConfigManager configManager;
+	private QuestHelperPanel questHelperPanel;
+
+	public AssistLevelPanel()
+	{
+		super(false);
+
+		setLayout(new BorderLayout());
+		setBackground(ColorScheme.DARK_GRAY_COLOR);
+
+		// Title
+		JPanel titlePanel = new FixedWidthPanel();
+		titlePanel.setLayout(new BorderLayout());
+
+		JLabel title = new JLabel();
+		title.setText("<html>Choose Assist Level</html>");
+		title.setForeground(Color.WHITE);
+		title.setBorder(BorderFactory.createEmptyBorder(10, 10, 0, 0));
+
+		titlePanel.add(title, BorderLayout.CENTER);
+
+		// Main content
+		mainPanel = new FixedWidthPanel();
+		mainPanel.setBorder(new EmptyBorder(8, 10, 10, 10));
+		mainPanel.setLayout(new DynamicGridLayout(0, 1, 0, 5));
+		mainPanel.setAlignmentX(Component.LEFT_ALIGNMENT);
+		mainPanel.add(titlePanel, BorderLayout.NORTH);
+
+		JPanel northPanel = new FixedWidthPanel();
+		northPanel.setLayout(new BorderLayout());
+		northPanel.add(titlePanel, BorderLayout.NORTH);
+		northPanel.add(mainPanel, BorderLayout.CENTER);
+
+		JScrollPane scrollPane = new JScrollPane(northPanel);
+		scrollPane.setHorizontalScrollBarPolicy(ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
+		add(scrollPane, BorderLayout.CENTER);
+	}
+
+	public void rebuild(QuestHelper questHelper, ConfigManager configManager, QuestHelperPanel questHelperPanel)
+	{
+		this.configManager = configManager;
+		this.questHelperPanel = questHelperPanel;
+
+		mainPanel.removeAll();
+
+		mainPanel.add(createFullAssistPanel(questHelper), BorderLayout.NORTH);
+		mainPanel.add(createMediumAssistPanel(questHelper), BorderLayout.CENTER);
+		mainPanel.add(createMinimumAssistPanel(questHelper), BorderLayout.SOUTH);
+	}
+
+
+	private JPanel createMinimumAssistPanel(QuestHelper questHelper)
+	{
+		JPanel minAssistPanel = new JPanel();
+		minAssistPanel.setLayout(new BorderLayout());
+
+		JLabel title = new JLabel();
+		title.setText("<html>Minimum Assistance</html>");
+		title.setForeground(Color.WHITE);
+		title.setBorder(BorderFactory.createEmptyBorder(10, 10, 0, 0));
+
+		JLabel description = new JLabel();
+		description.setText("<html>This disables most overlay-based help, and hides puzzle solutions. This is a good option if you just want the sidebar with helpful information, as it will still track where you are in the quest, and show items needed.</html>");
+		description.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
```

Comment:
Not a requirement by any means, but with this function added in Utils.java:
```java

	public JTextArea createStaticTextArea(String text)
	{
		var textArea = new JTextArea();
		textArea.setLineWrap(true);
		textArea.setWrapStyleWord(true);
		textArea.setOpaque(false);
		textArea.setEditable(false);
		textArea.setFocusable(false);
		textArea.setText(text);
		return textArea;
	}
```

You could replace this with
```suggestion
		var description = Utils.createStaticTextArea("Gives you the full helper experience, highlighting objects & dialogue options, puzzler solvers, and with arrows so you know where to go.");
		description.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
```

removing the need to use HTML to get text wrapping